app/controllers/v0/sessions_controller.rb

```diff
@@ -82,10 +82,22 @@ def destroy
     end
 
     def saml_logout_callback
-      if params[:SAMLResponse]
-        # We initiated an SLO and are receiving the bounce-back after the IDP performed it
-        handle_completed_slo
+      saml_settings = saml_settings(name_identifier_value: session&.uuid)
+      logout_response = OneLogin::RubySaml::Logoutresponse.new(params[:SAMLResponse], saml_settings, get_params: params)
+      logout_request  = SingleLogoutRequest.find(logout_response&.in_response_to)
+      session         = Session.find(logout_request&.token)
+      user            = User.find(session&.uuid)
+
+      errors = build_logout_errors(logout_response, logout_request, session, user)
+
+      if errors.size.positive?
+        extra_context = { in_response_to: logout_response&.in_response_to }
+        log_message_to_sentry("SAML Logout failed!\n  " + errors.join("\n  "), :error, extra_context)
       end
+      # in the future the FE shouldnt count on ?success=true
```

Comment:
@U-DON I changed the logic so you're always getting ?success=true, but could you make sure to not depend on that in the future for FE and we can remove that altogether?

Reply:
Yup, can remove it.

Reply:
But if I remove it now, won't it break the FE logic currently in production? It should be in future Phase 2 PR, right? just confirming.